gallery/src/styles/NxFormLayout/NxFormLayoutExample.tsx

```diff
@@ -36,25 +36,27 @@ export default function NxFormLayoutExample() {
     <form className="nx-form" onSubmit={onSubmit}>
       <div className="nx-form-group">
         <label className="nx-label">
-          <span className="nx-label__text">Label</span>
-          <NxStatefulTextInput validator={validator}/>
+          <span className="nx-label__text">A Field to Fill in</span>
+          <NxStatefulTextInput aria-required={true} validator={validator}/>
```

Comment:
Shouldn't this be outside the `<label>` to be consistent?

Reply:
> Shouldn't this be outside the <label> to be consistent?

No, the idea was to deliberately support both, since the old way is already used by consumers of RSC.

Reply:
> > Shouldn't this be outside the `<label>` to be consistent?
> 
> No, the idea was to deliberately support both, since the old way is already used by consumers of RSC.

Don't forget to update the docs to make it clear what the preferred layout is.